docs/generate-time-series/go.rst

```diff
@@ -0,0 +1,374 @@
+
+    func main() {
+        var err error
+        conn, err = pgx.Connect(context.Background(), "postgresql://crate@localhost:5432/doc")
+        if err != nil {
+            log.Fatalf("unable to connect to database: %v\n", err)
+        }
+	    defer conn.Close(context.Background())
+
+        conn.Exec(context.Background(), "CREATE TABLE iss (
+                                            timestamp TIMESTAMP GENERATED ALWAYS AS CURRENT_TIMESTAMP, 
+                                            position GEO_POINT)")
+    }
+
+
+Save all your changes and run the new code in the command line:
+
+.. code-block:: console
+
+    $ go run main.go
+
+When you run the script this time, the ``go`` command will look up the module 
+containing the `pgx`_ package and add it to ``go.mod``. In the `CrateDB 
+Admin UI`_, you should see the new table when you navigate to the *Tables* 
+screen using the left-hand navigation menu:
+
+.. image:: ../_assets/img/generate-time-series/table.png
+
+
+Record the ISS position
+=======================
+
+With the table in place, you can start recording the position of the ISS.
+
+Create some logic that calls your ``getISSPosition`` function and `INSERT`_ the
+result into the ``iss`` table:
+
+.. code-block:: js
+
+    ...
+
+    func main() {
+        ...
+
+        pos, err := getISSPosition()
+        if err != nil {
+            log.Fatalf("unable to get ISS position: %v\n", err)
+        } else {
+            _, err := conn.Exec(context.Background(), "INSERT INTO iss (position) VALUES ($1)", pos)
+            if err != nil {
+                log.Fatalf("unable to insert data: %v\n", err)
+            }
+        }
+    }
+
+Save all your changes and run the code in the command line:
+
+.. code-block:: console
+
+    $ go run main.go
+
+Press the up arrow on your keyboard and hit *Enter* to run the same command a
+few more times.
+
+When you're done, you can `SELECT`_ that data back out of CrateDB by executing
+``SELECT * FROM "doc"."iss"`` in the `SQL Console`_ of the `CrateDB Admin UI`_. 
+
+
+Automate the process
+====================
+
+Now that you have covered the key aspects, you can automate the data collection.
+
+In your file ``main.go``, create a function that encapsulates data insertion:
+
+.. code-block:: js
+
+    func insertData(position string) error {
+        _, err := conn.Exec(context.Background(), "INSERT INTO iss (position) VALUES ($1)", position)
+        return err
+    }
+
+Then in the ``main`` function of your script, create an infinite loop that 
+gets the latest ISS position and inserts the data into the database.  You 
+also have some basic error handling, in case either the API query or the
+CrateDB operation fails.  The script sleeps for 5 seconds after each sample. 
+Accordingly, the time series data will have a *resolution* of 5 seconds. If 
+you wish to change this resolution, you may want to configure your script 
+differently.
+
+Don't forget to import the `time`_ package. 
```

Comment:
like I mentioned in a previous comment, I am wondering if this would be easier to understand from the reader's pov if the code block came first and the explanation came after